### PR TITLE
Migrate back to Travis in docs

### PR DESCRIPTION
**Pull request by patcon**
Not sure why, but couldn't get scheduled tasks working on circleCI, so going back to travis as it was causing confusion.
(People had to click the CircleCI rebuild button manually, which is no one's job right now.)

Technically CircleCI is still active, but it's only running on push, so documentation should point to the more reliable place.

If anyone has experience getting scheduled tasks working on Circle, happy to flip back! Sorry for fragmentation of Travis

cc: @mr0grog @danielballan

### CODE REVIEW

**Comment by danielballan** [comment]
@patcon Did you ever reach out to Circle-CI support, following up on #20? Did they have any advice?

**Comment by Mr0grog** [comment]
I’m all but certain #20 was just an invalid config file. The unclear and inconsistent docs for Circle v2 are not a big help. :\

If I’m honest, I love how fast v2 is, but in absolutely all other respects it’s a huge step backwards. So I’m kinda ¯\\\_(ツ)\_/¯ on this change.

**Comment by Mr0grog** [comment]
FWIW, this is the error message I see in Circle’s “workflows” panel for this project:

screen shot 2018-02-14 at 10 10 57 am

```
Config does not conform to schema:
{
  workflows {
    :nightly {
      :triggers [
        {
          :schedule {
            :filters missing-required-key
          }
        }
      ]
    }
  }
}
```

…which makes sense, since this branch (which I think is what triggered that build/error?) is missing that key. This looks like it was branched off from before I fixed the config:
- Master (my fix from #20): https://github.com/edgi-govdata-archiving/edgi-scripts/blob/master/.circleci/config.yml#L43-L54
- This PR https://github.com/edgi-govdata-archiving/edgi-scripts/blob/1bc85914c971503057603074032dcdb4aed55314/.circleci/config.yml#L43-L50

**Comment by patcon** [comment]
haha was juuuuussst about to give up after another hour of messing with this, but just now got it working (at least partially!)

tl;dr - when you add a workflow key with a scheduled workflow, there is now no default per-commit workflow. that was throwing me off and frustrating my efforts to troubleshoot.

So I can now confirm that the job is working. I'll check back tonight and see whether it ran. If so, I'll switch back the docs :)

https://circleci.com/gh/edgi-govdata-archiving/workflows

**Comment by dcwalk** [comment]
Hey @patcon does the fact that CircleCI works mean we can revert this PR? 🎉

**Comment by patcon** [comment]
heh yep, just got to that in https://github.com/edgi-govdata-archiving/edgi-scripts/issues/26 :)

we are so in sync! (plus or minus several months sometimes)
